setup.py

```diff
@@ -68,11 +68,17 @@ def install_requires():
     setup(
         name=DISTNAME,
         version=VERSION,
-        packages=find_packages(),
+        packages=find_packages(exclude=["tests"]),
         entry_points=ENTRY_POINTS,
         python_requires=">=3.5",
-        install_requires=install_requires(),
-        package_data={"": ["README.md", "LICENSE.txt"]},
+        use_scm_version=True,
+        setup_requires=['setuptools_scm'],
```

Comment:
maybe needs a comment explaining that this is what makes `include_package_data` work

Reply:
Let's give it a try like this I will release a dummy version and see how it goes.